lib/TabTracker.js

```diff
@@ -0,0 +1,89 @@
+const tabs = require("sdk/tabs");
+const ss = require("sdk/simple-storage");
+
+const NEWTAB_URL = "resource://activity-streams/data/content/activity-streams.html#/";
+
+const TabTracker = {
+  _openTabs: [],
+
+  init: function() {
```

Comment:
If you need an init function, have you considered making TabTracker an object?

i.e.
```javascript
function TabTracker(options = {}) {
 ...
};

TabTracker.prototype = {
 ...
}
```
That would also obviate the need to do the function binding: `this.onOpen = this.onOpen.bind(this);`